components/engine/engine-odata/src/test/java/org/eclipse/dirigible/components/odata/transformers/OData2ODataMTransformerTest.java

```diff
@@ -321,372 +321,5 @@ public void testAggregationsTransformation() throws IOException, SQLException {
         assertArrayEquals(new String[] {aggregationEntity}, transformed);
     }
 
-    // @Test
```

Comment:
@delchev @ThuF do you remember why we've commented this?

Reply:
Let's either try to fix the tests or leave it as it is.

Reply:
I've just reverted this removal